certified-connectors/Peltarion/apiDefinition.swagger.json

```diff
@@ -47,34 +47,6 @@
         "operationId": "callapi",
         "x-ms-visibility": "important",
         "parameters": [
-          {
-            "name": "code",
-            "in": "query",
-            "required": true,
-            "type": "string",
-            "default": "FHWJvOJnrrBz8v7DOWoABLS1K2e9Im6q7YalAthQMGkqTa57jDYm/w==",
-            "x-ms-visibility": "internal",
-            "description": "Authentication for the API proxy",
-            "x-ms-summary": "This is a key required by the API proxy"
-          },
-          {
-            "name": "peltarionurl",
-            "in": "header",
-            "required": true,
-            "type": "string",
-            "x-ms-visibility": "important",
-            "description": "Peltarion URL",
-            "x-ms-summary": "URL to identify your Peltarion API"
-          },
-          {
-            "name": "peltariontoken",
-            "in": "header",
-            "required": true,
-            "type": "string",
-            "x-ms-visibility": "important",
-            "description": "Peltarion token",
-            "x-ms-summary": "Token to identify your Peltarion API"
-          },
           {
             "name": "peltarionbody",
             "in": "header",
```

Comment:
@andvra The summary and description says it is body parameter, but definition says "in":"header". If it is body parameter that expects json object, it should have "type":"object".

Reply:
Our platform expects body tags with a number of different structures, so it won't match a specific schema, which is required if we send this as a body. Therefore, we stringify the body and send it as a header parameter.